nibe/connection/nibegw.py

```diff
@@ -564,8 +609,25 @@ def _encode(self, obj, context, path):
             ),
         ),
         "RMU_WRITE_REQ": Struct(
```

Comment:
Why we need RMU_WRITE_REQ request? Isn't it duplication of MODBUS_WRITE_REQ functionality?
How you are plannig to use it?

Reply:
I dont have a full plan for it at the moment. I just like to have as much of the protocol documented somewhere.

It is nice to replay a trace and see what is going on when there are actual RMUs installed.